scripts/build-all.sh

```
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+## 一键打包所有 package
+
+# 获取 yarn dev/build 类型
+buildType=build
+if [ -n "$1" ]; then  
+  buildType=$1
+fi
+
+cd ./packages
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)